cumulusci/tasks/salesforce/communities_preflights.py

```diff
@@ -0,0 +1,21 @@
+import requests
+from cumulusci.tasks.salesforce import BaseSalesforceApiTask
+
+
+class IsCommunitiesEnabled(BaseSalesforceApiTask):
+    api_version = "48.0"
+
+    def _run_task(self):
+        s = requests.Session()
+        s.get(self.org_config.start_url).raise_for_status()
+        r = s.get(
+            "{}/sites/servlet.SitePrerequisiteServlet".format(
+                self.org_config.instance_url
+            )
+        )
+        self.return_values = r.status_code == 200
```

Comment:
What status code is returned if communities are not enabled? Should we distinguish between that specific status code and other codes like 503 which could indicate inability to complete the check?

Reply:
It returns a 500 when Communities are not enabled, but that could potentially mean other things too (this API is not documented).

As far as we know, any status code other than 200 means that a first Community-create call won't be successful - this is the endpoint that the UI calls when an org's first Community creation is initiated, and it continues to return 200 for subsequent calls.

I might have found a better proxy for Community availability, though. It looks like the `Network` sObject doesn't get added to the schema until Communities are explicitly turned on, even if the licenses are available in the org. I'll revise to use that strategy instead.